docs/cndb_omf.md

```diff
@@ -0,0 +1,294 @@
+DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!! DRAFT!!!
+
+# OMF descriptions
+
+Describes the proposed CNDB omf for HSE-3.0.
+
+## Record types
+
+Each record type contains a header at the beginning. This header contains the type of the record and its
+size. This header is not listed in the following breakdown of the various record types.
+
+### Version
+- Magic: A magic number to verify that the MDC is a CNDB MDC.
+- Version: Version of the CNDB omf.
+- Captgt: (Capacity Target) Size of the CNDB.
+
+### Info
+There are two types of info records. `CNDB_TYPE_INFO` and `CNDB_TYPE_INFOD`.
+- Create-time parameters for a KVS
+  - Prefix length: Length of prefixes when using prefix delete.
+- Flags: Flags for the KVS
+  - `CN_CFLAG_CAPPED`: Whether or not the KVS is a capped KVS.
+- Meta size: The Info record is followed by opaque metadata. This field records the size of this metadata.
```

Comment:
Does the order of these matter? For reading purposes, it might make sense to group this near the Meta[] since the CN Name is a fixed a length.